Patches/Caffe/Patch.cmake

```diff
@@ -1,6 +1,20 @@
-
-file(COPY
-  ${Caffe_patch}/Dependencies.cmake
-  DESTINATION
-  ${Caffe_source}/cmake/
-  )
+if(WIN32)
+	file(COPY
+	  ${Caffe_patch}/WindowsDownloadPrebuiltDependencies.cmake
+	  DESTINATION
+	  ${Caffe_source}/cmake/)
+	file(COPY
+	  ${Caffe_patch}/FindGFlags.cmake
+	  DESTINATION
+	  ${Caffe_source}/cmake/modules/)
+	file(COPY
+	  ${Caffe_patch}/CMakeLists.txt
+	  DESTINATION
+	  ${Caffe_source}/)
+else()
+	file(COPY
+	  ${Caffe_patch}/Dependencies.cmake
+	  DESTINATION
+	  ${Caffe_source}/cmake/
+	  )
```

Comment:
Maybe the indents don't need to change then if that's how the code was in caffe.